src/main/scala-0.27/parsley/XCompat.scala

```diff
@@ -0,0 +1,13 @@
+package parsley
+
+import scala.collection.mutable
+
+private[parsley] object XCompat {
```

Comment:
This is a direct copy of 2.13's.  We could DRY that up with additional build complexity, but let's not until this gets much bigger.